index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/b08094b2a2.js" crossorigin="anonymous"></script>
+    <title>Personal Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav><i class="fas fa-bars"></i></nav>
+    </header>
+    <section>
+        <h1 class="header">Hey there.
+            <br>I’m Ian.
+           <h2 class="text_color">I’m a software developer</h2>
+        </h1>
+        <p class="paragraph"> I can help you build a product , <br>feature or website Look through <br>some of my work and
+            experience! <br>If you like
+            what you see and have <br> a project you need coded,
+            <br>don’t hesitate to contact me.
+        </p>
+
+        <div>
+            <ul class="images">
+                <li><a href=""><img src="img/twitter.png" alt="Twitter"></a></li>
+                <li> <a href=""><img src="img/Linkedin icon.png" alt="Linkedin"></a></li>
+                <li><a href=""> <img src="img/medium.png" alt="Medium"></a></li>
+                <li><a href=""><img src="img/github.png" alt="Github"></a></li>
+                <li><a href=""><img src="img/angellist.png" alt="Angellist"></a></li>
+            </ul>
+        </div>
+    </section>
```

Comment:
Kindly adjust the headline section to match the design. If it's pretty and works, it'll be a wonderful website.
Note the following:
- The background image seems to be repeating unlike in the design.
- Add more margin above the "Hey there" text
- The circled arrow below the social icons seems to be missing
- Some of the designs of the background image are missing

Your design:
![image](https://user-images.githubusercontent.com/20554583/235425466-c8839df2-c874-4cc0-9369-7b191df10bc1.png)    ![image](https://user-images.githubusercontent.com/20554583/235425477-a9a51be9-6c29-4695-8d7f-5b4b61f78737.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/235425512-6056ad06-69a5-4fc6-b5aa-b9e779849dca.png)